src/ElmHtml/Query.elm

```diff
@@ -96,15 +98,39 @@ queryAll selectors =
     query (Multiple selectors)
 
 
+{-| Query a Html element using a selector, without considering
+any descendants lower than its immediate children.
+-}
+queryChildren : Selector -> ElmHtml -> List ElmHtml
+queryChildren =
+    queryInNodeHelp False
+
+
+{-| Query to ensure a html node has all selectors given, without considering
+any descendants lower than its immediate children.
+-}
+queryChildrenAll : List Selector -> ElmHtml -> List ElmHtml
+queryChildrenAll selectors =
+    queryInNodeHelp False (Multiple selectors)
+
+
 {-| Query a Html node using a selector
 -}
 queryInNode : Selector -> ElmHtml -> List ElmHtml
-queryInNode selector node =
+queryInNode =
+    queryInNodeHelp True
+
+
+queryInNodeHelp : Bool -> Selector -> ElmHtml -> List ElmHtml
+queryInNodeHelp recurse selector node =
```

Comment:
:art: I think recurse isn't a very clear name - it doesn't explain what it does. `shouldLookInChildren` might be clearer